opentracing/src/main/java/io/opentracing/Tracer.java

```diff
@@ -87,15 +87,18 @@
       /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
       SpanBuilder withTag(String key, String value);
 
-      /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
+      /** Same as {@link Span#setTag(String, boolean)}, but for the span being built. */
       SpanBuilder withTag(String key, boolean value);
 
-      /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
+      /** Same as {@link Span#setTag(String, Number)}, but for the span being built. */
       SpanBuilder withTag(String key, Number value);
 
-      /** Specify a timestamp of when the Span was started, represented in microseconds since epoch. */
+      /** Same as {@link Span#setBaggageItem(String, String)}, but for the span being built. */
```

Comment:
Line 96 needs to be swapped with 99, will resubmit.

Line 90, 93 were incorrect.